src/app/presentations/[slug]/page.tsx

```diff
@@ -0,0 +1,37 @@
+import Link from 'next/link';
+
+import { getIndexData } from '@/models/get-index-data';
+
+export default async function PresentationBySlug({ params }: { params: { slug: string } }) {
+  const { presentations } = await getIndexData();
+  const result = presentations.find((item) => {
+    return item.slug === params.slug || null;
```

Comment:
kell ide a  `|| null` ?

Reply:
illetve érdemes lehet egy checket csinálni, hogy létezik-e result. Ha nem, csak kiírsz annyit, hogy ez az előadás nem található, meg egy link, hogy vissza az előadásokhoz. Így egyrészt nem kell kérdőjelezgetni később, és random url-ekre is értelmes hibaüzenetet kap a felhasználó